CHANGELOG.md

```diff
@@ -4,14 +4,15 @@
 This version includes:
 
 - support for omero prefix
-- support for floating images with very small pixels range
+- improved manipulation of rendering setting for floating images with very small pixels range
 - support for maximum intensity projection along Z
 - load of ROIS using the web api
-- add shortcut to draw line
-- improve copy and paste of ROIs
-- add support for new Lookup tables
-- ability to open several images
-- improve layout of rois table
+- added ability to draw line
+- improved copy and paste of ROIs and add a new context menu for these actions
+- added support for new Lookup tables
+- added ability to open multiple selected images
+- improved layout of rois table
```

Comment:
maybe rois -> ROIs  (unify with the capitalized version three lines above this one)